joomla/40-backend-template
# form validation fixes

Pull Request for Issue # .

### Summary of Changes
Invalid form fields should indicate that are invalid (when loosing focus, e.g. onBlur)

### Testing Instructions
Edit an article, click on the title input and then click on the alias input. The Title input should become red!
Enter something on the title input and click on the alias, see the green border
Invalid
![screen shot 2016-12-29 at 14 06 56](https://cloud.githubusercontent.com/assets/3889375/21543567/4e5b13da-cdd0-11e6-93c6-86574ca64103.png)
Valid
![screen shot 2016-12-29 at 14 16 47](https://cloud.githubusercontent.com/assets/3889375/21543708/93c308f0-cdd1-11e6-9d8b-49d8be1087db.png)

This works only on fields that have a tag REQUIRED!!!

B/C BREAK: Removed the old label notation that was used before Bootstrap (Mootools era)!!!
It was about time to go... 

### Documentation Changes Required
